srcCs/PerftestTransport.cs

```diff
@@ -1033,6 +1097,17 @@ private void ConfigureSecurityFiles(DDS.DomainParticipantQos qos)
 
             return true;
         }
+
+        public string getMulticastAddr(string topicName)
+        {
+            string ret;
```

Comment:
apply same changes as in cpp
address instead of ret

Reply:
Done.

Reply:
Done